src/main/resources/main.ts

```diff
@@ -64,7 +65,7 @@ runAsSu(() => {
     },
     {
       name: "import-result-contributors",
-      enabled: true,
+      enabled: !notNullOrUndefined(app.config.contributors),
```

Comment:
How about this? It makes it more explicit what we are doing?

Can you also add documentation to _README.md_ about this config option?

```suggestion
      enabled: app.config.importResultContributors !== "disabled",
```